wger/manager/templates/schedule/overview.html

```diff
@@ -20,11 +20,7 @@
     {% for schedule in schedules %}
     <tr>
         <td>
-            <a href="{% url 'manager:schedule:view' schedule.id %}">{{ schedule.name }}</a>
-
-            {% if schedule.is_active %}
-            <span class="label label-primary">{% trans "active" %}</span>
-            {% endif  %}
+            <a href="{% url 'manager:schedule:view' schedule.id %}">{{ schedule.name }}</a>&nbsp;{% if schedule.is_active %}<span class="label label-primary">{% trans "active" %}</span>{% endif  %}
```

Comment:
Is it necessary to make the ifs one liners? Shoudn't the nbsp make space in any case? Other than that, the PR is ready